client/app/dbaas/logs/detail/aliases/logs-aliases.service.js

```diff
@@ -0,0 +1,223 @@
+        this.LogStreamsConstants = LogStreamsConstants;
+        this.LogAliasConstants = LogAliasConstants;
+        this.UrlHelper = UrlHelper;
+    }
+
+    /**
+     * returns array of aliases with details
+     *
+     * @param {any} serviceName
+     * @returns promise which will be resolve to array of aliases. Each alias will have all details populated.
+     * @memberof LogsAliasesService
+     */
+    getAliases (serviceName) {
+        return this.getAliasesDetails(serviceName)
+            .catch(this.ServiceHelper.errorHandler("logs_aliases_get_error"));
+    }
+
+    /**
+     * gets details for each alias in array
+     *
+     * @param {any} serviceName
+     * @returns promise which will be resolve to an array of alias objects
+     * @memberof LogsAliasesService
+     */
+    getAliasesDetails (serviceName) {
+        return this.getAliasesIds(serviceName)
+            .then(aliases => {
+                const promises = aliases.map(aliasId => this.getAapiAlias(serviceName, aliasId));
+                return this.$q.all(promises);
+            });
+    }
+
+    /**
+     * returns array of aliases id's of logged in user
+     *
+     * @param {any} serviceName
+     * @returns promise which will be resolve to array of aliases id's
+     * @memberof LogsAliasesService
+     */
+    getAliasesIds (serviceName) {
+        return this.AliasApiService.query({ serviceName }).$promise;
+    }
+
+    /**
+     * returns details of an alias
+     *
+     * @param {any} serviceName
+     * @param {any} aliasId
+     * @returns promise which will be resolve to alias object
+     * @memberof LogsAliasesService
+     */
+    getAlias (serviceName, aliasId) {
+        return this.AliasApiService.get({ serviceName, aliasId })
+            .$promise.catch(this.ServiceHelper.errorHandler("logs_alias_get_error"));
+    }
+
+    /**
+     * returns details of an alias
+     *
+     * @param {any} serviceName
+     * @param {any} aliasId
+     * @returns promise which will be resolve to alias object
+     * @memberof LogsAliasesService
+     */
+    getAapiAlias (serviceName, aliasId) {
+        return this.AliasAapiService.get({ serviceName, aliasId })
+            .$promise.catch(this.ServiceHelper.errorHandler("logs_alias_get_error"));
+    }
+
+    /**
+     * returns objecy containing total number of aliases and total number of aliases used
+     *
+     * @param {any} serviceName
+     * @returns quota object containing V (total number aliases) and configured (number of aliases used)
+     * @memberof LogsAliasesService
+     */
+    getQuota (serviceName) {
+        return this.AccountingAapiService.me({ serviceName }).$promise
+            .then(me => {
+                const quota = {
+                    max: me.total.maxNbAlias,
+                    configured: me.total.curNbAlias,
+                    currentUsage: me.total.curNbAlias * 100 / me.total.maxNbAlias
+                };
+                return quota;
```

Comment:
it's a detail so don't feel obliged to fix this => 
return {
    max: ...,
    configured: ...
}

Don,t declared a variable if you do nothing with it :)

Reply:
removed it